src/app/frontend/replicationcontrollerdetail/replicationcontrollerdetail.html

```diff
@@ -27,103 +29,20 @@ <h1 flex class="md-title kd-replicationcontrollerdetail-app-name">
-            <span ng-if="::ctrl.replicationControllerDetail.podInfo.running" class="kd-comma-separated-item">
-              {{::ctrl.replicationControllerDetail.podInfo.running}} running
-            </span>
-          </span>
-        </div>
-        <span class="kd-replicationcontrollerdetail-sidebar-line">Label selector</span>
-        <span class="kd-replicationcontrollerdetail-sidebar-subline kd-has-labels">
-          <kd-labels labels="::ctrl.replicationControllerDetail.labelSelector"></kd-labels>
-        </span>
-        <span class="kd-replicationcontrollerdetail-sidebar-line">Labels</span>
-        <span class="kd-replicationcontrollerdetail-sidebar-subline kd-has-labels">
-          <kd-labels labels="::ctrl.replicationControllerDetail.labels"></kd-labels>
-        </span>
-        <span class="kd-replicationcontrollerdetail-sidebar-line">Images</span>
-        <span class="kd-replicationcontrollerdetail-sidebar-subline"
-              ng-repeat="image in ctrl.replicationControllerDetail.containerImages">
-          {{image}}
-        </span>
-        <div ng-if="::ctrl.replicationControllerDetail.services" layout="column">
-          <span class="kd-replicationcontrollerdetail-sidebar-title">Services</span>
-          <div ng-repeat="service in ::ctrl.replicationControllerDetail.services">
-            <span class="kd-replicationcontrollerdetail-sidebar-line">Name</span>
-            <span class="kd-replicationcontrollerdetail-sidebar-subline">
-              {{::service.name}}
-            </span>
-            <span class="kd-replicationcontrollerdetail-sidebar-line">Label selector</span>
-            <span class="kd-replicationcontrollerdetail-sidebar-subline kd-has-labels">
-              <kd-labels labels="::service.selector"></kd-labels>
-            </span>
-            <span class="kd-replicationcontrollerdetail-sidebar-line">Internal endpoint</span>
-            <div class="kd-replicationcontrollerdetail-sidebar-subline">
-              <div ng-if="::service.internalEndpoint">
-                <kd-service-endpoint endpoint="::service.internalEndpoint"></kd-service-endpoint>
-              </div>
-              <span ng-hide="service.internalEndpoint">none</span>
-            </div>
-            <span class="kd-replicationcontrollerdetail-sidebar-line">External endpoint</span>
-            <div class="kd-replicationcontrollerdetail-sidebar-subline">
-              <div ng-repeat="endpoint in service.externalEndpoints">
-                <kd-service-endpoint endpoint="::endpoint"></kd-service-endpoint>
-              </div>
-              <span class="kd-replicationcontrollerdetail-sidebar-subline"
-                  ng-hide="::service.externalEndpoints">
-                none
-              </span>
-            </div>
-          </div>
-        </div>
-      </div>
+    <kd-replication-controller-info details="::ctrl.replicationControllerDetail">
+    </kd-replication-controller-info>
+  </div>
+  <div flex>
```

Comment:
Btw, can you add html formatter to the project? 
https://www.npmjs.com/package/gulp-prettify

This would save us time on formatting disputes. To me, it already works awesome for JavaScript, so it should for HTML.